include/zephyr/kernel.h

```diff
@@ -375,7 +375,7 @@ FUNC_NORETURN void k_thread_user_mode_enter(k_thread_entry_t entry,
  * @param ... list of kernel object pointers
  */
 #define k_thread_access_grant(thread, ...) \
-	FOR_EACH_FIXED_ARG(k_object_access_grant, (;), thread, __VA_ARGS__)
+	FOR_EACH_FIXED_ARG((k_object_access_grant), (;), (thread), __VA_ARGS__)
```

Comment:
This one isn't correct.  "k_object_access_grant" is a symbol name used verbatim in the macro, it's not a macro parameter that is going to be expanded and subject to precedence pollution.